src/app/stylesheets/custom/custom-impac.less

```diff
@@ -0,0 +1,16 @@
+// This file includes style changes that overwrite some of the defaults in impac-angular
+
+.content.accounts-balance {
```

Comment:
This will only modify the account balance widget.

Also I think we should have a look in the impac-angular project if there is no distinct background color vars for the widget header and body. If not, create those variables.